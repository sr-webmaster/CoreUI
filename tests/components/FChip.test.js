import { mount } from '@vue/test-utils'
import Component from '@freshinup/core-ui/src/components/FChip.vue'

describe('FChip', () => {
  it('snapshot default', async () => {
    const wrapper = mount(Component)
    expect(wrapper.element).toMatchSnapshot()
  })
  it('snapshot with color', async () => {
    const wrapper = mount(Component, {
      propsData: {
        color: 'info'
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
