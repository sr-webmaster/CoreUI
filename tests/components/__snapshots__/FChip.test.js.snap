// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FChip snapshot default 1`] = `
<span
  class="v-chip theme--light undefined-text--text"
  tabindex="0"
>
  <span
    class="v-chip__content"
  />
</span>
`;

exports[`FChip snapshot with color 1`] = `
<span
  class="v-chip theme--light info info-text--text"
  tabindex="0"
>
  <span
    class="v-chip__content"
  />
</span>
`;
